frontend/src/APIClients/BaseAPIClient.ts

```diff
@@ -17,15 +18,19 @@ baseAPIClient.interceptors.request.use(
   async (config: AdaptAxiosRequestConfig) => {
     const newConfig = { ...config };
 
+    let authHeaderParts = [""];
     // if access token in header has expired, do a refresh
-    const authHeaderParts = [""];
+    if (typeof config.headers.Authorization === "string") {
```

Comment:
Just curious, is this check necessary / was it introduced due to a bug in the original code? Would we not be able to simplify this code to just:

`const authHeaderParts = config.headers.Authorization?.split(" ");`?

Reply:
I believe the code was initially config.headers.Authorization?.split(" "), but the type of Authorization can be a number or some other value now (with the previous dependency updates) and so now to split the value we need to check if its a string.